it/repositories/RegistrationRepositorySpec.scala

```diff
@@ -87,9 +90,26 @@ class RegistrationRepositorySpec extends AnyFreeSpec
 
     "must return None when no registration exists" in {
 
-      val result = repository.get(RegistrationData.registration.vrn).futureValue
+      val result = repository.get(registration.vrn).futureValue
 
       result must not be defined
     }
   }
+
+  ".set" - {
+
+      "must update registration Date of First Sale" in {
+
+        val currentRegistration = registration copy (dateOfFirstSale = None, lastUpdated = Instant.now(stubClock))
```

Comment:
We probably dont want to update the lastUpdate on the current registration as that is what we use for the record coming back from the db. So this will still have the original last updated time. That actually gets updated as part of our method that we are testing